src/@resources/databaseUser/inputs.schema.json

```diff
@@ -21,4 +21,4 @@
       "type": "object"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
This happens a lot in your PRs and clutters the diffs. Do you have different vs code settings on or something?

Reply:
No there should not be, I am honestly unsure why this is different. My VSCode is setup using the same settings we have for all our other projects.